examples/ads.amp.html

```diff
@@ -641,6 +642,15 @@ <h2 id="flite">Flite</h2>
       data-mixins="">
   </amp-ad>
 
+  <h2 id="fluct">fluct</h2>
+  <amp-ad width="300" height="250"
+      type="fluct"
+      data-g="1000067784"
+      data-u="1000101409">
+    <div placeholder></div>
```

Comment:
with default placeholder supported. no need to add placeholder and fallback here.